content/trailmap/prologue.fr.md

```diff
@@ -0,0 +1,143 @@
++++
+title = "Dogecoin Trailmap: Prologue"
+date = "2021-12-10"
+type = "about"
+[ author ]
+name = "Dogecoin Fondation"
++++
+
+<section class="presentation">
+<div class="left">
+
+<div class="title">
+
+
+ ## <img width="60px" style='display: inline;' src="/marker.png"/>Dogecoin Trailmap: Prologue
+
+<div class="underline"></div>
+</div>
+
+<div class="description">
+ 
+* [Dogecoin Trailmap: Prologue](fr/trailmap/prologue/) 
+* [Projet: Dogecoin.com makeover & Dogepedia](fr/trailmap/website/)
+* [Projet: Libdogecoin](fr/trailmap/libdogecoin/)
+* [Projet: Dogecoin Standard](fr/trailmap/standard/)
+* [**Projet: GigaWallet**](fr/trailmap/gigawallet/)
+* Projet: Dogecoin Keyring (App / SDK)
+* Projet: RadioDoge (Starlink + Radio nodes pour région éloignée)
+* Proposition: Community Staking (Proof of Stake)
+* Partenariat: Point de vente (L2 app) 
+</div>
+
+<div class="right">
+<img class="dogegoin-light" src="/trailmap.png" alt="Dogecoin logo">
+<img class="dogegoin-dark" src="/trailmap.png" alt="Dogecoin logo">
+</div>
+
+
+</section>
+
+<section class='board'>
+
+## Qu'est-ce que le Dogecoin Trailmap ?
+  
+Bien que "feuille de route" soit un terme couramment utilisé lorsque de nouvelles crypto-monnaies sont lancées, il évoque une voie rigide à suivre, dirigée par une seule entreprise ou "leader".  
+
+  **Dogecoin est un projet open-source basé sur le consensus et dirigé par la communauté et sa direction est formée par les nombreux contributeurs individuels et organisationnels aux différents projets de la blockchain et de l'écosystème Dogecoin.**  
+
+La direction de Dogecoin est pavée par les nombreux projets que la communauté 
+estime qu'ils correspondent le mieux à sa direction. C'est une décision très intentionnelle : il est important que
+Dogecoin reste un projet décentralisé avec des bergers plutôt que des dirigeants, chacun des projets et les contributions se tiennent ou échouent sur leurs propres mérites.
+
+Cette feuille de route représente une promenade que la Fondation Dogecoin s'est engagée à suivre et qui, selon nous, contribueront au succès de Dogecoin.
+
+
+## Pourquoi ces projets ?
+
+Au mois d'août, nous avons signé le [Dogecoin Manifesto](fr/manifeste) et invité la communauté à le signer en cliquant sur le bouton de partage de tweet : la réponse a été 
```

Comment:
missing '/' before 'fr', it should be '/fr/xxxxx';

and 'manifeste' should be 'manifesto' here as it's a URL path